### PR TITLE
update project data for Pallets

### PR DESCRIPTION
**Pull request by davidism**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -13,78 +14,44 @@ Install and update using `pip`_:
...
-
-  $ git clone git@github.com:pallets/flask-sqlalchemy.git
-  $ cd flask-sqlalchemy
-  $ tox
-
-That will run everything, so you  might want to start with something like `tox -e py37` or whatever
-version of Python you have installed.
 
-You can figure out our testing and documentation dependencies by further inspecting the `tox.ini`
-file in the root of the project.
```
Comment (rsyring) [inline README.rst]: I think it's worth keeping this section as it makes it easier for potential contributors to know how to get started.  Or, if not here, then maybe add it to the docs.

File: CHANGES.rst [hunk 2]
```diff
@@ -1,198 +1,194 @@
-Changelog
-=========
-
-
```
Comment (rsyring) [inline CHANGES.rst]: If you want to add the changelog contents from #715, then #715 can just be closed as the other changes in it are made mute by this PR.

File: README.rst (hunk 1 above)
Comment (davidism) [reply README.rst]: I'm working on standard contributing docs for Pallets.

File: CHANGES.rst (hunk 2 above)
Comment (davidism) [reply CHANGES.rst]: cherry picked your changelog commit

**Pull request merged**